test/gurka/test_traffic.cc

```diff
@@ -110,7 +110,6 @@ TEST(Traffic, BasicUpdates) {
                "it's noticed the changes in the live traffic file"
             << std::endl;
   {
-
```

Comment:
the test below this one was failing at first because i blocked an edge and its opp edge if edge was hard filtered. the problem with that is that traffic closures can close only one direction of an edge which means i needed to check both directions separately. which you'll see in search.cc